spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/workflow/instance/BackingServicesUpdateValidatorService.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.workflow.instance;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import reactor.core.publisher.Mono;
+
+import org.springframework.cloud.appbroker.deployer.BackingService;
+
+/**
+ * Validates updates to backing services, and returns an updated copy of the list with the previousPlan set. This help
+ * only triggering plan update when previousPlan is set.
+ */
+public class BackingServicesUpdateValidatorService {
+
+	// Accept and return Mono to make it easier for AppDeploymentUpdateServiceInstanceWorkflow to provide us with previous and candidate backing services
```

Comment:
Remove the comment, please

Reply:
thanks

Reply:
fixed in 661f7ed3e6ef425dfdd7baabd18f96f6b4d7c287